packages/components/src/index.tsx

```diff
@@ -170,4 +176,26 @@ const CourierComponents: React.FunctionComponent = () => {
     </CourierProvider>,
     courierRoot
   );
-})();
+
+  hasInit = true;
+};
+
+window.courierSdk = {
```

Comment:
We might need a more clear separation of concerns about what happens on the window and what happens through react state(and reducers)